package.json

```diff
@@ -64,7 +64,7 @@
   },
   "devDependencies": {
     "@types/dateformat": "^3.0.1",
-    "@types/react": "^16.9.43",
+    "@types/react": "^16.14.4",
```

Comment:
❓ Why is this change required? Does it need to be applied to all projects?

Reply:
This should be discarded 😓